scripts/contract-upgrade-governance.sh

```diff
@@ -44,6 +44,12 @@ EOF
 exit 1
 }
 
+# Check if guardiand command exists. It's needed for generating the protoxt and
+# computing the digest.
+if ! command -v guardiand >/dev/null 2>&1; then
```

Comment:
`command -v` is totally fine, but `type -p` is a shell builtin and does the same thing